centos-7.template

```diff
@@ -120,37 +120,44 @@ base64 -d < cert-gen.sh.base64 > cert-gen.sh
 chmod +x cert-gen.sh
 mv cert-gen.sh /opt
 
-# Workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1414250
-ln -s /usr/bin/dockerd-current /usr/bin/dockerd
-
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1414250 is fixed and now latest version doesn't need synlink